shards/test-ipfs.sh

```diff
@@ -4,9 +4,9 @@ set -e
 # INSTALL IPFS
 apt-get update
 apt-get install -y wget
-wget -q https://dist.ipfs.tech/kubo/v0.16.0/kubo_v0.16.0_linux-amd64.tar.gz
-tar -xvzf kubo_v0.16.0_linux-amd64.tar.gz
-export PATH=$PATH:`pwd`/kubo
+wget -q https://dist.ipfs.tech/go-ipfs/v0.12.0/go-ipfs_v0.12.0_linux-amd64.tar.gz
```

Comment:
kubo 0.17, let's do a try with that too? or doesn't work?

Reply:
It doesn't work with kubo 0.17.

Reply:
That's so weird to be honest but must be libp2p incompatibilities somehow ( a library engineered for compatibility 😄 )